docs/kmesh_deploy_and_develop_in_kind.md

```diff
@@ -121,12 +121,14 @@ You can follow the steps below to develop in kind:
     You can change the logger level by:
 
     ```shell
-    kubectl exec -it $kmesh_pod_name -n kmesh-system -- kmesh-daemon log --set default:debug
+    ./kmeshctl log $kmesh_pod_name --set default:debug
     ```
 
+   `make build` will generate kmeshctl in the kmesh directory.
```

Comment:
After v0.5.0 release, `kmeshctl` will be included in the release tarball, and we could use it directly without compiling it on site.

Reply:
I have removed it.